dotnet-nunit/SauceLabs.Visual.Example/SauceDemo.cs

```diff
@@ -65,7 +65,15 @@ public async Task SauceDemo_CheckInventory_ShouldAddBackpackToCart()
         await VisualClient.VisualCheck("Inventory Page",
             new VisualCheckOptions()
             {
-                IgnoreElements = new[] { addToBackpack },
+                DisableOnly = new [] { DiffingOption.Position },
+                Regions = new []
+                {
+                    new SelectiveRegion()
+                    {
+                        Element = addToBackpack,
+                        EnableOnly = new [] { DiffingOption.Content },

```

Comment:
Now that I see it here, shouldn't we use an enum with `[Flags]` [attribute](https://dev.to/ayodejii/enum-flags-in-c-20a6)?

Reply:
Yup, It has been changed !